docs/getting-started.md

```diff
@@ -69,12 +76,13 @@ Only when everything is working are you ready to package the application. In thi
           tags:
             - foo
             - bar
-1. Before packaging up into a container, try running it with `dry-run` and `verbose` options. If you're using the atomicapp command use `atomicapp --dry-run --verbose run`. This should output the commands that will run. Common errors:
+
+1. Before packaging up into a container, try running it in a test mode if one is provided by your Nulecule implementation.  If you are using the [Atomic App reference implemenation](https://github.com/projectatomic/atomicapp), use the `dry-run` and `verbose` options as follows: `atomicapp --dry-run --verbose run`. This should output the commands that will run. Common errors:
     * provider files don't match the artifact relative path
     * yaml or json is not valid
     * missing parameter
 
-1. Once the Nulecule file and provider artifacts are working, package the application as a container. Typically, this means basing it off of an executable image such as `projectatomic/atomicapp`. Unless you have a special use case, the stock Dockerfile may be used unaltered.
+1. Once the Nulecule file and provider artifacts are working, package the application as a container. Typically, this means basing it off of an executable image provided by the implemenation of Nulecule you are using.  If you are using the [Atomic App reference implemenation](https://github.com/proje  ctatomic/atomicapp), the stock Dockerfile may be used unaltered unless you have a special use case.
```

Comment:
Atomic App which is the reference implementation of the Nulecule Specification ?! or just remove 'reference implementation'